core/Tracker/Visit.php

```diff
@@ -179,6 +184,7 @@ public function handle()
         }
 
         $isNewVisit = $this->request->getMetadata('CoreHome', 'isNewVisit');
+        $this->previousVisitProperties = new VisitProperties($this->request->getMetadata('CoreHome', 'lastKnownVisit') ?: []);
```

Comment:
btw did something similar in https://github.com/matomo-org/matomo/pull/15869/files#diff-8a4e24a5a8b6afb6ee880dddb4934c43R114

could actually maybe reuse that logic and have eg `visitProperties->getOriginalProperty($column)` which then looks for a value in `visitInfo[self::KEY_ORIGINAL_VISIT_ROW][$column]` . Or maybe we refactor `self::KEY_ORIGINAL_VISIT_ROW` to use same approach as you implemented if possible as we have eg access to the visitor object here: https://github.com/matomo-org/matomo/pull/15869/files#diff-73996192b2fb6075808b45d6276c3264R72  but would need to tweak https://github.com/matomo-org/matomo/pull/15869/files#diff-8a4e24a5a8b6afb6ee880dddb4934c43R131 to also get the `Visitor` object instead of visit properties. I reckon your solution is likely nicer 👍